client/my-sites/plan-features/index.jsx

```diff
@@ -468,6 +483,7 @@ export default connect(
 				const newPlan = isNew( plan ) && ! isPaid;
 				const currentPlan = sitePlan && sitePlan.product_slug;
 				const showMonthlyPrice = ! relatedMonthlyPlan && showMonthly;
+				const currentPlanMatch = !! ( sitePlan && ( getPlanClass( sitePlan.product_slug ) === getPlanClass( plan ) ) );
```

Comment:
@tyxla I tried a few selector implementations and I abandonded them. #14635 is one. Do you think that's worth considering, or better for `lib/plans`?